README.md

```diff
@@ -47,6 +47,26 @@ You can use Alfred to
 
 `command + d` : when given "don't save, cancel, and save", triggers "don't save"; also short for saving something to the desktop
 
+## Terminal / Bash
+
+`ctrl + a` : jump to the beginning of line
+
+`ctrl + u` : clear line
+
+`↑` and `↓` : cycle through command history 
+
+`!!` : last command, e.g. `sudo !!`, `!! > log.txt`
+
+`|` : piping warrants its own section
```

Comment:
I agree. With that in mind I would rather not place it here without its proper explanation.